packagemanifests/0.0.1/antrea-operator-for-kubernetes.clusterserviceversion.yaml

```diff
@@ -1,351 +0,0 @@
-apiVersion: operators.coreos.com/v1alpha1
```

Comment:
I think another pr is handling this?

Reply:
Indeed, for convenience I've rebased this change on top of PR #51